_health-care/_js/_components/date_input.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import _ from 'lodash';
+
+class DateInput extends React.Component {
+  constructor() {
+    super();
+    this.state = {month: '', date: '', year: ''};
+    this.handleMonthChange = this.handleMonthChange.bind(this);
+    this.handleDateChange = this.handleDateChange.bind(this);
+    this.handleYearChange = this.handleYearChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.id = _.uniqueId('date-input-');
```

Comment:
Oh, is this going to affect our navigation? Can you not go back and edit responses?